core/src/main/resources/hudson/model/Api/index.jelly

```diff
@@ -24,13 +24,13 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
-	<l:layout title="Remote API">
-		<l:main-panel>
+  <l:layout title="Remote API" type="one-column">
+    <l:main-panel>
       <h1>REST API</h1>
 
       <p>
         Many objects of Jenkins provide the remote access API. They are available
-        at <tt>${rootURL}/.../api/</tt> where "..." portion is the object for
+        at <code>${rootURL}/.../api/</code> where "..." portion is the object for
```

Comment:
@oleg-nenashev We could consider this pieces of text as _code_ https://developer.mozilla.org/en/docs/Web/HTML/Element/code